src/miral/symbols.map

```diff
@@ -482,3 +482,10 @@ global:
 
 } MIRAL_4.0;
 
+MIRAL_4.2 {
+global:
+  extern "C++" {
+    miral::X11Support::X11Support*;
```

Comment:
This needs more hackery: their are two identical globs that match both constructors